packages/kokoas-e2e/cypress.config.ts

```diff
@@ -1,9 +1,50 @@
+import { getAllInvoices } from 'api-kintone';
+import { saveInvoices } from 'api-kintone/src/invoice/saveInvoices';
 import { defineConfig } from 'cypress';
 
+const baseUrl = process.env.KT_REDIRECT_URI;
+
 export default defineConfig({
   projectId: 'iemq2k',
   e2e: {
-    experimentalRunAllSpecs : true,
-    baseUrl: 'https://rdmuhwtt6gx7.cybozu.com/k/176/#',
+    experimentalRunAllSpecs: true,
+    baseUrl: `${baseUrl}#`,
+    setupNodeEvents(on) {
+      on('task', {
+        async prepareInvoice(chkDataId: string) {
+          const recInvoices = (await getAllInvoices())
+            .filter(({ invoiceStatus, estimateLists }) => {
+              if ((invoiceStatus.value === 'sent') 
+              || (invoiceStatus.value === 'created')) {
+                return estimateLists.value.some(({ value }) => {
+                  return value.dataId.value === chkDataId;
+                });
+              }
+            });
+
+          recInvoices.forEach(async ({ uuid }) => {
+            return saveInvoices({
+              recordId: uuid.value,
+              record: {
+                invoiceStatus: { value: 'voided' },
+              },
+            });
+          });
+
+          return recInvoices.length;
+        },
+      });
+      on('task', {
+        async prepareInvoiceReissue(eissueInvoiceId: string) {
```

Comment:
処理を見る限り、以下の関数名は適切だと思います。

```suggestion
        async setInvoiceStatusToSent(eissueInvoiceId: string) {
```

もし、利用目的を拡大するなら、setInvoiceStatus(invoiceId, status) でもいいと思います。

Reply:
ご指摘ありがとうございます。
とりあえずは実装内容は変更しませんでした。
[02dc948](https://github.com/Lorenzras/yumecoco-monorepo/pull/173/commits/02dc948207553e1435c64097aa07fc6b0f25d47e)にて修正しました。